CONTRIBUTING.md

```diff
@@ -1,6 +1,34 @@
 # Contributing
 
-We love pull requests. If you have something you want to add or remove, please open a new pull request.
+If you have something you want to add or remove, please use the following agreed upon process.
+
+### Initial Proposal Creation
+ - Proposals are made by creating a GitHub [issue](https://help.github.com/articles/creating-an-issue/) or [pull request](https://help.github.com/articles/creating-a-pull-request/) (PR).
+ - Issues containing the proposal are tagged with the area(s) under which they fall using [GitHub labels](https://github.com/sul-dlss/DeveloperPlaybook/labels).
+ - The proposal is an appropriate granularity requirement to capture existing consensus.
+ - The proposal is of the form When <doing activity>, we [MUST|SHOULD] <use described approach>.
```

Comment:
I don't think that was necessarily the intent of that statement (although, looking at the issues raised so far, we've already deviated from that practice). I think the goal was to provide a strong statement of what problem a proposal is designed to address, e.g.

- "When developing projects, we must use version control"
- "When releasing code, we must apply a license"
- "When documenting best practices, we should link to examples of those practices being used"

Further, in https://github.com/sul-dlss/DeveloperPlaybook/pull/29/files (which, I guess, is blocked by this?), I believe we explicitly address your concern (perhaps prompted by your concerns elsewhere?):

> The goals are explicitly NOT to [...] Mandate or set in stone any unbreakable requirements. They are expectations that can be deviated from for good, documented reasons.